source/frontend/src/features/mapSideBar/disposition/tabs/offersAndSale/OffersAndSaleContainer.tsx

```diff
@@ -0,0 +1,62 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useDispositionProvider } from '@/hooks/repositories/useDispositionProvider';
+import {
+  Api_DispositionFile,
+  Api_DispositionFileOffer,
+  Api_DispositionFileSale,
+} from '@/models/api/DispositionFile';
+
+import { IOffersAndSaleContainerViewProps } from './OffersAndSaleContainerView';
+
+export interface IOffersAndSaleContainerProps {
+  dispositionFile?: Api_DispositionFile;
+  View: React.FC<IOffersAndSaleContainerViewProps>;
+}
+
+const OffersAndSaleContainer: React.FunctionComponent<IOffersAndSaleContainerProps> = ({
+  dispositionFile,
+  View,
+}) => {
+  const {
+    getDispositionFileOffers: { execute: getDispositionFileOffers, loading: loadingOffers },
+    getDispositionFileSales: { execute: getDispositionFileSales, loading: loadingSales },
+  } = useDispositionProvider();
+  const [dispositionOffers, setDispositionOffers] = useState<Api_DispositionFileOffer[]>([]);
+  const [dispositionSale, setDispositionSale] = useState<Api_DispositionFileSale | null>(null);
+
+  const fetchDispositionInformation = useCallback(async () => {
+    if (dispositionFile?.id) {
+      const dispositionOffersPromise = getDispositionFileOffers(dispositionFile?.id);
+      const dispositionSalesPromise = getDispositionFileSales(dispositionFile?.id);
+
+      const [offersResponse, salesResponse] = await Promise.all([
+        dispositionOffersPromise,
+        dispositionSalesPromise,
+      ]);
+
+      if (offersResponse) {
+        setDispositionOffers(offersResponse);
```

Comment:
nit: could simplify this to just get the data object that is returned by the api request wrapper instead of needing two additional state vars.

Reply:
I am using state vars to hold the list of offers and  be able to update the offers state once the delete offers succeds.